From 6d3770b7a42c89b0014b3db52cac0ab3c4f4111a Mon Sep 17 00:00:00 2001
From: Bogdan Purcareata <bogdan.purcareata@nxp.com>
Date: Fri, 9 Feb 2018 15:01:29 +0200
Subject: [PATCH 367/448] staging: fsl-dpaa2/eth: Remove Kconfig dependency on QBMAN_DEBUG

The DPIO implementation no longer uses this Kconfig option. DPAA2 eth
debugfs entries are dependent on this, so remove it.

Signed-off-by: Bogdan Purcareata <bogdan.purcareata@nxp.com>
---
 drivers/staging/fsl-dpaa2/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/Kconfig b/drivers/staging/fsl-dpaa2/Kconfig
index 8665d3a..6b6dd86 100644
--- a/drivers/staging/fsl-dpaa2/Kconfig
+++ b/drivers/staging/fsl-dpaa2/Kconfig
@@ -29,7 +29,7 @@ config FSL_DPAA2_ETH_USE_ERR_QUEUE
 
 # QBMAN_DEBUG requires some additional DPIO APIs
 config FSL_DPAA2_ETH_DEBUGFS
-	depends on DEBUG_FS && FSL_QBMAN_DEBUG
+	depends on DEBUG_FS
 	bool "Enable debugfs support"
 	default n
 	---help---
-- 
1.7.1

